src/nodes/Network/IPFSSubscribe.js

```diff
@@ -17,52 +16,41 @@ IPFSSub.title = "IPFSSubscribe";
 
 IPFSSub.prototype.onReceiveMsg = async function(msg) {
   console.log(msg)
-  this.message = msg.data.toString()
+  this.message = Buffer.from(msg.data).toString('utf-8')
   console.log("this.message is now",this.message)
   this.trigger("received",this.message)
 }
 
 IPFSSub.prototype.onAdded = async function() {
   this.title_color = "#dddddd";
-  this.ipfs = new IPFS({
-    EXPERIMENTAL: {
-     pubsub: true
+  this.ipfs = await IPFS.create({
+   EXPERIMENTAL: {
+     pubsub: true,
    },
    repo: 'ipfs-' + Math.random(),
    config: {
-     Addresses: {
-       Swarm: ['/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star']
-     }
+      Addresses: {
+        Swarm: ['/dns4/wrtc-star1.par.dwebops.pub/tcp/443/wss/p2p-webrtc-star',
+        '/dns4/wrtc-star2.sjc.dwebops.pub/tcp/443/wss/p2p-webrtc-star']
+     },
+     Bootstrap: []
    }
   })
-  await this.ipfs.ready
-  console.log('IPFS (subscribe) node is ready')
   const { id, agentVersion, protocolVersion } = await this.ipfs.id()
-  console.log("IPFS!")
-  const res = await this.ipfs.bootstrap.list()
-  console.log("IPFS",res)
+  console.log("IPFS FOR SUBSCRIBE!", id, agentVersion, protocolVersion)
   console.log("IPFS SUBSCRIBING TO ",this.properties.channel)
-  await this.ipfs.pubsub.subscribe(this.properties.channel, this.onReceiveMsg.bind(this))
+  this.ipfs.pubsub.subscribe(this.properties.channel, this.onReceiveMsg.bind(this))
   console.log("IPFS SUBSCRIBED")
   this.title_color = "#eeee44";
   this.status = "connected"
 };
 
 IPFSSub.prototype.onExecute = function() {
   let channel = this.getInputData(0)
-  if(channel && this.properties.channel!=channel){
+  if(channel && this.properties.channel!==channel){
       this.onPropertyChanged("channel",channel)
   }
   this.setOutputData(0,this.message)
-  //console.log(this.ipfs)
-  if(this.ipfs && this.ipfs.isOnline() && this.ipfs.pubsub && typeof this.ipfs.pubsub.peers == "function"){
-    const peerCount = this.ipfs.pubsub.peers()
-    if( peerCount>0){
-      this.status = peerCount+" peers"
-      this.title_color = "#ee4444";
-    }
-  }
-
```

Comment:
Don't see any logic that adds peers, so I took this out. If we need to revisit this to add another functionality or I am not seeing it correctly then feel free to let me know.